### PR TITLE
Routing and Components

### PR DESCRIPTION
**Pull request by znjs**
- Routing for all pages
- Components that will be used on pages

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -46,6 +46,6 @@
 
 .links a {
   color: #d9d9d9;
-  font-size: 1.2rem;
+  /* font-size: 1.2rem; */
```
Comment (shobit1337) [inline src/App.css]: Remove unnecessory commented code.

File: src/components/Hero/hero.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.hero-container {
+    /* background-image: url("/assets/hero1.jpg"); */
```
Comment (shobit1337) [inline src/components/Hero/hero.css:2]: Remove unnecessory commented code.

File: src/components/NavBar/nav.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    flex-grow: 1;
+}
+
+nav {
+    height: 4rem;
+}
+
+@media only screen and (max-width:1052px) {
+
+    /* nav>div:nth-child(2) {
```
Comment (shobit1337) [inline src/components/NavBar/nav.css:19]: Remove unnecessory commented code.

File: src/components/Products/Products.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useFilter } from "../../context";
+import { ProductCard } from "../ProductCard/ProductCard";
+
+function Products() {
+  const { productsState } = useFilter();
+  // console.log(productsState.products);
```
Comment (shobit1337) [inline src/components/Products/Products.jsx]: Remove unnecessory commented code.

**Review by Ash-D23 — APPROVED** [review]
The basic routing setup looks good used NavLinks

File: src/pages/Authentication/auth.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
+:root {
+    --transparent-white-29: #f9fafb29;
+    --transparent-white-40: #f9fafb40;
+    --blend-color: #b98c6e;
+}
+
+.auth-page {
+    /* ?BACKGOUND IMAGE */
+    background-color: var(--blend-color);
+    /* background-image: url("/assets/christmas.jpg"); */
```
Comment (shobit1337) [inline src/pages/Authentication/auth.css:10]: Remove unnecessory commented code.

**Review by shobit1337 — APPROVED** [review]
Your code is very well structured and clean code. Great work @znjs

File: src/backend/db/products.js [hunk 6]
```diff
@@ -1,30 +1,35 @@
...
+      }),
+      itemDesc:
+        "lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem ",
+      imageUrl: `https://picsum.photos/200/300?random=${index}`,
+      altText: "Item Image",
+      discount: true,
+      wishlisted: false,
+      cart: false,
+      saveForLater: false,
+      outOfStoack: false,
```
Comment (muskaanshah) [inline src/backend/db/products.js:29]: typo: outofstock

**Review by muskaanshah — APPROVED** [review]

**Pull request merged**